app/password/new/controller.js

```diff
@@ -0,0 +1,31 @@
+import Ember from "ember";
+import EmberValidations from "ember-validations";
+
+export default Ember.Controller.extend(EmberValidations.Mixin, {
+  hasSubmitted: false,
+
+  validations: {
+    'model.password': {
+      presence: true,
+      confirmation: { message: 'does not match password' },
+      'password-complexity': true
+    }
+  },
+
+  actions: {
+
+    save: function(model){
+      var hasSubmitted = this.get('hasSubmitted');
+      if (!hasSubmitted) {
+        this.set('hasSubmitted', true);
+      }
+      this.validate().then(() => {
+        this.get('target').send('save', model);
+      }).catch(function(){
+        // Silence the validation exception
```

Comment:
no. That error would occur in a different frame of JavaScript and not be caught here.